circleci/table_circleci_build.go

```diff
@@ -0,0 +1,96 @@
+			{Name: "is_first_green_build", Description: "Indicates if this is the first succeeded build of the project.", Type: proto.ColumnType_BOOL},
+			{Name: "lifecycle", Description: "Build lifecycle.", Type: proto.ColumnType_STRING},
+			{Name: "outcome", Description: "Result of the build.", Type: proto.ColumnType_STRING},
+			{Name: "parallel", Description: "Number of parallel execution.", Type: proto.ColumnType_INT},
+			{Name: "platform", Description: "Platform version where build ran.", Type: proto.ColumnType_STRING},
+			{Name: "previous", Description: "Previous build.", Type: proto.ColumnType_JSON},
+			{Name: "queued_at", Description: "Timestamp of when the build was queued.", Type: proto.ColumnType_STRING},
+			{Name: "reponame", Description: "Repository bane.", Type: proto.ColumnType_STRING},
+			{Name: "retries", Description: "Number of build retries.", Type: proto.ColumnType_JSON},
+			{Name: "retry_of", Description: "Precedent build of the retrial.", Type: proto.ColumnType_INT},
+			{Name: "ssh_users", Description: "SSH users with access to the build, if any.", Type: proto.ColumnType_JSON, Transform: transform.FromField("SSHUsers")},
+			{Name: "start_time", Description: "Start time of the build.", Type: proto.ColumnType_TIMESTAMP},
+			{Name: "status", Description: "Status of the build.", Type: proto.ColumnType_STRING},
+			{Name: "stop_time", Description: "Stop time of the build.", Type: proto.ColumnType_TIMESTAMP},
+			{Name: "subject", Description: "Commit message that triggered the build.", Type: proto.ColumnType_STRING},
+			{Name: "timedout", Description: "Indicates if the build got timedout.", Type: proto.ColumnType_BOOL},
+			{Name: "usage_queued_at", Description: "Usage queued time.", Type: proto.ColumnType_STRING},
+			{Name: "user", Description: "User who triggered the build to run.", Type: proto.ColumnType_JSON},
+			{Name: "username", Description: "Organization username.", Type: proto.ColumnType_STRING},
+			{Name: "vcs_revision", Description: "VCS Revision.", Type: proto.ColumnType_STRING},
+			{Name: "vcs_tag", Description: "VCS Tag.", Type: proto.ColumnType_STRING},
+			{Name: "vcs_url", Description: "VCS URL.", Type: proto.ColumnType_STRING, Transform: transform.FromField("VCSURL")},
+			{Name: "workflow", Description: "Workflow which ran the build.", Type: proto.ColumnType_JSON, Transform: transform.FromField("workflows")},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listCircleciBuilds(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	client, err := ConnectV1Sdk(ctx, d)
+	if err != nil {
+		logger.Error("circleci_build.listCircleciBuilds", "connect_error", err)
+		return nil, err
+	}
+
+	limit := -1
+	offset := 0
+
+	builds, err := client.ListRecentBuilds(limit, offset)
+	if err != nil {
+		logger.Error("circleci_build.listCircleciBuilds", "list_builds_error", err)
+		if strings.Contains(err.Error(), "Not found") {
+			return nil, nil
+		}
```

Comment:
```suggestion
		if isNotFoundError(err) {
			return nil, nil
		}
```

Since we have defined `isNotFoundError()` in the `errors.go` file, we should use the function to check for `Not found` error code.

Reply:
@luisffc could you please make this change across all the tables?